Do not install clFFT/clBLAS headers.

Patch by Marius Brehler.

--- src/backend/opencl/CMakeLists.txt
+++ src/backend/opencl/CMakeLists.txt
@@ -18,9 +18,6 @@ ADD_DEFINITIONS(-DAF_OPENCL
 FIND_PACKAGE(CLBLAS REQUIRED)
 INCLUDE_DIRECTORIES(${CLBLAS_INCLUDE_DIRS})
 LINK_DIRECTORIES(${CLBLAS_LIBRARY_DIR})
-# Custom install command
-INSTALL(DIRECTORY "${CLBLAS_PACKAGE_DIR}/include/" DESTINATION include 
-    COMPONENT headers)
 IF(UNIX)
     FILE(GLOB CLBLAS_LIBS "${CLBLAS_PACKAGE_DIR}/lib64/*${CMAKE_SHARED_LIBRARY_SUFFIX}*")
 ELSEIF(WIN32)
@@ -33,9 +30,6 @@ INSTALL(FILES ${CLBLAS_LIBS} DESTINATION
 FIND_PACKAGE(clFFT REQUIRED)
 INCLUDE_DIRECTORIES(${CLFFT_INCLUDE_DIRS})
 LINK_DIRECTORIES(${CLFFT_LIBRARY_DIR})
-# Custom install command
-INSTALL(DIRECTORY "${CLFFT_BUILD_DIR}/package/include/" DESTINATION include
-    COMPONENT headers)
 IF(UNIX)
   FILE(GLOB CLFFT_LIBS "${CLFFT_BUILD_DIR}/package/lib64/*${CMAKE_SHARED_LIBRARY_SUFFIX}*")
 ELSE(UNIX)
